cartoframes/keys.py

```diff
@@ -0,0 +1,56 @@
+"""API key utility functions
+"""
+import os
+import six
+import warnings
+
+if not six.PY3:
+    FileNotFoundError = OSError
```

Comment:
good question. It's a python 2/3 compatibility hack. In python 2, using `os.remove` ([line 53](https://github.com/CartoDB/cartoframes/pull/84/files#diff-08b66c83e0a8c2676019c56d8d3b481dR53)) on a file that doesn't exist raises `OSError`. On python 3, it raises `FileNotFoundError`.